### DISCUSSION TITLE
View (not saving) tables/fields name as capitals

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#4378] [Feature Requests]
I am creating some tables that are a single word, but the tables/fields are acronyms, such as IEEE, after I create a table or fields it will be Ieee, so it is not that nice visually. Perhaps is a question of 80/20 but could be  something so simple as adding a configuration to the table and field such as capitalised name but from the view not actually from the database.

What do you think?

**Comment by nickrum** [comment]
You could add translations to the table/field names to have prettier names for all user facing occurrences of those names.

**Reply by benhaynes** [reply]
@nickrum is correct — the primary way to handle edge-cases is through Collection/Field translations. If the capitalization is non-proprietary and/or has extremely popular usage, then we can add it to Core via our `format-title` package (in other words, PRs welcome 😄 ).

https://github.com/directus/directus/tree/main/packages/format-title/src

**Reply by juancarlosjr97** [reply]
Thank you @nickrum and @benhaynes :) 

Cool, I would be happy to do that change then, I think I have the idea how to add it. I will work on the PR, thank you guys!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
